v0/pipeline/steps/ci/aqua-trivy-step-info.yaml

```diff
@@ -16,7 +16,7 @@ allOf:
       enum:
       - default
     image:
-      $ref: stoyaml-image.yaml
+      $ref: ../../stages/ci/stoyaml-image.yaml
```

Comment:
step module should not be dependent on any file under stage. Move stoyaml-image.yaml to steps/ci